autofish.lua

```diff
@@ -0,0 +1,220 @@
+-- handles automatic fishing jobs to limit the number of fish the fortress keeps on hand
+-- autofish [enable | disable] <max> [min] [--include-raw | -r]
+
+--@ enable=true
+
+local json = require("json")
+local persist = require("persist-table")
+local argparse = require("argparse")
+local dump = require("dumper")
+
+local GLOBAL_KEY = "autofish"
+
+local help = [====[
+autofish
+=============
+Manages your fish stocks by toggling fishing labours as you reach certain stock thresholds.
+
+
+Usage
+-------------
+  autofish [enable | disable | status] | <max> [min] [--include-raw | -r]
```

Comment:
I'd suggest:
```
enable autofish
autofish [status]
autofish <max> [<min>] [<options>]
```

we're trying to standardize on `enable X` instead of `X enable` so we have a consistent interface for the upcoming `gui/control-panel`. It's ok to still support the `X enable` syntax, but we've been avoiding documenting that to promote usage of the `enable` interface.